src/vulnerabilityreports/page.tsx

```diff
@@ -126,19 +124,21 @@ function renderScanner(scanner: Scanner) {
 
 function renderImage(report: VulnerabilityReport) {
     const imageRef = report.getImageRef()
+    // drop registry from displayed image reference; keep in tooltip
+    const shortImageRef = imageRef.substring(imageRef.indexOf('/')+1)
```

Comment:
We use this extension also for debugging Starboard so removing parts of names or tooltips might be confusing. Let's propose such changes in issues or dedicated PR.